chunkator/__init__.py

```diff
@@ -11,9 +11,12 @@ def chunkator(queryset, chunk_size):
     length. We're saving at least a ``count()`` query on QuerySets, or a
     CPU-and-RAM-consuming ``len(queryset)`` query.
     """
-    pk = 0
+    pk = None
     while True:
-        page = queryset.filter(pk__gt=pk).order_by('pk')[:chunk_size]
+        queryset = queryset.order_by('pk')
```

Comment:
I would like to take a concrete example. 

Onboarding. I don't want to iterate over my available tasks ordering by the PK, but by the Order of the Tasks inside the Process. It's a functionnal requirement, how can I do that with your order overriding ?